### ISSUE TITLE
Questions about manually partitioning data sets

### ISSUE THREAD

**Issue by liangwenge740** [ecker-lab/TreeLearn#21]
First of all, I would like to thank you for your work, which has been of great help to my research. I would like to ask if I can use 75% of the regions in the L1W data as the training set for training and the remaining 25% as the test set? If feasible, which hyperparameters in the yaml file need to be adjusted?

**Comment by jonaden94** [comment]
Hello, in principle you can use whatever data you want for training, validation and testing. What you have to do:

1. decide on which data to use for training, validation and testing (potentially splitting point clouds e.g. using cloudcompare)
2. Put the point clouds in the respective folders required for training/validation/testing. For this, just take a look at ``configs/data_gen/gen_val_data.yaml``, ``configs/data_gen/gen_train_data.yaml`` and ``pipeline/pipeline.yaml``

The readme is also quite comprehensive. It basically contains all the information.

Final remark: There are also many other labeled datasets out there (e.g. FOR-Instance, LautX and Wytham Woods). I do not think it is necessary anymore to use the same point cloud for training/validation/testing.
